runtime/flang/fpcvt.c

```diff
@@ -31,6 +31,17 @@ union ieee {
   int i[2];
 };
 
+union ieeeq {
+  long double q;
```

Comment:
isn't the assumption on `long double` size too optimistic here?

Reply:
I can replace this (and other uses of `long double`) with `__float128` which should be supported by GCC and Clang on aarch64, powerpc64, and x86_64. But I don't have anyway to test powerpc64.